app/index.js

```diff
@@ -10,7 +10,7 @@ import routes from './routes';
 import '../node_modules/bootstrap/dist/css/bootstrap.min.css';
 import '../node_modules/toastr/build/toastr.min.css';
 import '../node_modules/griddle-react-bootstrap/dist/griddle-react-bootstrap.css';
-import '../ux/dev/css/styles.css';
+// import '../ux/dev/css/styles.css'; //ToDo: commenting MSUIX styles until fully integrated.
```

Comment:
No probs! I have a different opinion but I see your point. This is fine -- thanks!